File: _persist/temp/venda/Java/Transpagto.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("TRANSPAGTO")
public class Transpagto extends CollectionItem
{
    @Campo("ID_TRANSACAO", CampoTipo.tfKey)
    public String Id_Transacao;
    @Campo("NR_SEQ", CampoTipo.tfKey)
    public Integer Nr_Seq;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("ID_CAIXA", CampoTipo.tfReq)
    public Integer Id_Caixa;
    @Campo("TP_DOCUMENTO", CampoTipo.tfReq)
    public Integer Tp_Documento;
    @Campo("ID_HISTREL", CampoTipo.tfReq)
    public Integer Id_Histrel;
    @Campo("NR_PARCELA", CampoTipo.tfReq)
    public Integer Nr_Parcela;
    @Campo("NR_PARCELAS", CampoTipo.tfReq)
    public Integer Nr_Parcelas;
    @Campo("NR_DOCUMENTO", CampoTipo.tfNul)
    public Integer Nr_Documento;
    @Campo("VL_DOCUMENTO", CampoTipo.tfReq)
    public Float Vl_Documento;
    @Campo("DT_VENCIMENTO", CampoTipo.tfReq)
    public DateTime Dt_Vencimento;
    @Campo("CD_AUTORIZACAO", CampoTipo.tfNul)
    public String Cd_Autorizacao;
    @Campo("NR_NSU", CampoTipo.tfNul)
    public Integer Nr_Nsu;
    @Campo("DS_REDETEF", CampoTipo.tfNul)
    public String Ds_Redetef;
    @Campo("NM_OPERADORA", CampoTipo.tfNul)
    public String Nm_Operadora;
    @Campo("NR_BANCO", CampoTipo.tfNul)
    public Integer Nr_Banco;
    @Campo("NR_AGENCIA", CampoTipo.tfNul)
    public Integer Nr_Agencia;
    @Campo("DS_CONTA", CampoTipo.tfNul)
    public String Ds_Conta;
    @Campo("NR_CHEQUE", CampoTipo.tfNul)
    public Integer Nr_Cheque;
    @Campo("DS_CMC7", CampoTipo.tfNul)
    public String Ds_Cmc7;
    @Campo("TP_BAIXA", CampoTipo.tfNul)
    public Integer Tp_Baixa;
    @Campo("CD_OPERBAIXA", CampoTipo.tfNul)
    public Integer Cd_Operbaixa;
    @Campo("DT_BAIXA", CampoTipo.tfNul)
    public DateTime Dt_Baixa;
}

File: _persist/temp/venda/CSharp/Transpagto.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TRANSPAGTO")]
    public class Transpagto : CollectionItem
    {
        [Campo("ID_TRANSACAO", CampoTipo.tfKey)]
        public string Id_Transacao { get; set; }
        [Campo("NR_SEQ", CampoTipo.tfKey)]
        public int Nr_Seq { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("ID_CAIXA", CampoTipo.tfReq)]
        public int Id_Caixa { get; set; }
        [Campo("TP_DOCUMENTO", CampoTipo.tfReq)]
        public int Tp_Documento { get; set; }
        [Campo("ID_HISTREL", CampoTipo.tfReq)]
        public int Id_Histrel { get; set; }
        [Campo("NR_PARCELA", CampoTipo.tfReq)]
        public int Nr_Parcela { get; set; }
        [Campo("NR_PARCELAS", CampoTipo.tfReq)]
        public int Nr_Parcelas { get; set; }
        [Campo("NR_DOCUMENTO", CampoTipo.tfNul)]
        public int Nr_Documento { get; set; }
        [Campo("VL_DOCUMENTO", CampoTipo.tfReq)]
        public double Vl_Documento { get; set; }
        [Campo("DT_VENCIMENTO", CampoTipo.tfReq)]
        public DateTime Dt_Vencimento { get; set; }
        [Campo("CD_AUTORIZACAO", CampoTipo.tfNul)]
        public string Cd_Autorizacao { get; set; }
        [Campo("NR_NSU", CampoTipo.tfNul)]
        public int Nr_Nsu { get; set; }
        [Campo("DS_REDETEF", CampoTipo.tfNul)]
        public string Ds_Redetef { get; set; }
        [Campo("NM_OPERADORA", CampoTipo.tfNul)]
        public string Nm_Operadora { get; set; }
        [Campo("NR_BANCO", CampoTipo.tfNul)]
        public int Nr_Banco { get; set; }
        [Campo("NR_AGENCIA", CampoTipo.tfNul)]
        public int Nr_Agencia { get; set; }
        [Campo("DS_CONTA", CampoTipo.tfNul)]
        public string Ds_Conta { get; set; }
        [Campo("NR_CHEQUE", CampoTipo.tfNul)]
        public int Nr_Cheque { get; set; }
        [Campo("DS_CMC7", CampoTipo.tfNul)]
        public string Ds_Cmc7 { get; set; }
        [Campo("TP_BAIXA", CampoTipo.tfNul)]
        public int Tp_Baixa { get; set; }
        [Campo("CD_OPERBAIXA", CampoTipo.tfNul)]
        public int Cd_Operbaixa { get; set; }
        [Campo("DT_BAIXA", CampoTipo.tfNul)]
        public DateTime Dt_Baixa { get; set; }
    }
}